threatstream_connector.py

```diff
@@ -1829,8 +1911,10 @@ def _handle_detonate_file(self, param):
             "report_radio-classification": param.get('classification')
         })
         if param.get("use_premium_sandbox", None) and param.get("use_vmray_sandbox", None):
-            return action_result.set_status(phantom.APP_ERROR,
-                "Both premium sandbox and vmray sandbox cannot be selected simultaneously for detonation. Please select one of them.")
+            return action_result.set_status(
+                phantom.APP_ERROR,
+                "Both premium sandbox and vmray sandbox cannot be selected simultaneously for detonation. Please select one of them")
```

Comment:
"Premium sandbox and vmray sandbox cannot be selected simultaneously for detonation. Please select one of them")

Reply:
Included the suggested change.